Config/Project64.rdb

```diff
@@ -8133,4 +8113,4 @@ Good Name=MGC 2011 Demo (PD)
 Internal Name=shut up and code
 CPU Type=Interpreter
 RDRAM Size=8
-Status=Unsupported
+Status=Unsupported
```

Comment:
@dsx- Other than adding another line break at the end, I see nothing wrong with merging this.

Maybe change this:

Rat Attack
Core Note=Interpreter only for multiplayer

Other than that, it looks great. :smile: